sematic/ui/src/types/Types.tsx

```diff
@@ -541,14 +545,81 @@ function PlotlyFigureValueView(props: ValueViewProps) {
 
 function MatplotlibFigureValueView(props: ValueViewProps) {
   let { valueSummary } = props;
-  let { path } = valueSummary;
+
+  const hasFigureJsonData = useMemo(() => !!valueSummary['mpld3'], [valueSummary]);
+  
+  return hasFigureJsonData ? 
+    <MatplotlibFigureValueFigure key={valueSummary['mpld3']['id']} spec={valueSummary['mpld3']} />
+    : <MatplotlibFigureValueImage path={valueSummary['path']} /> ;
+}
+
+interface MatplotlibFigureValueImageProps{
+  path: string
+}
+
+function MatplotlibFigureValueImage(props: MatplotlibFigureValueImageProps) {
```

Comment:
This React component takes care of the old format (in case there is still any left in DB)